### PR TITLE
{toolchains} GCC 9.1.0 + binutils 2.32

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/55c8f6ec0136e81bd86f717afd554317 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 10 out of 10 (8 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/35975209e2e6a66e51566150236a369f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2424.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0e0e5d1031ea0b96805268876b4cf95f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2016.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cf448790b2d8676becdff8a13bc0f10b for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 0 out of 8 (8 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/410f482e6caf881d1a6d10b1f9faba90 for a full test report.

File: easybuild/easyconfigs/b/Bison/Bison-3.3.2-GCCcore-9.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'Bison'
+version = '3.3.2'
+
+homepage = 'http://www.gnu.org/software/bison'
```
Comment (akesandgren) [inline easybuild/easyconfigs/b/Bison/Bison-3.3.2-GCCcore-9.1.0.eb]: https ??

File: easybuild/easyconfigs/b/binutils/binutils-2.32-GCCcore-9.1.0.eb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name = 'binutils'
+version = '2.32'
+
+homepage = 'http://directory.fsf.org/project/binutils/'
```
Comment (akesandgren) [inline easybuild/easyconfigs/b/binutils/binutils-2.32-GCCcore-9.1.0.eb]: https ?

File: easybuild/easyconfigs/g/GCC/GCC-9.1.0-2.32.eb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'Bundle'
+
+name = 'GCC'
+version = '9.1.0'
+
+binutilsver = '2.32'
+versionsuffix = '-%s' % binutilsver
+
+homepage = 'http://gcc.gnu.org/'
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/GCC/GCC-9.1.0-2.32.eb]: https ?

File: easybuild/easyconfigs/g/GCCcore/GCCcore-9.1.0.eb [hunk 4]
```diff
@@ -0,0 +1,49 @@
+easyblock = 'EB_GCC'
+
+name = 'GCCcore'
+version = '9.1.0'
+
+homepage = 'http://gcc.gnu.org/'
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-9.1.0.eb]: https ?

File: easybuild/easyconfigs/m/M4/M4-1.4.18-GCCcore-9.1.0.eb [hunk 5]
```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'M4'
+version = '1.4.18'
+
+homepage = 'http://www.gnu.org/software/m4/m4.html'
```
Comment (akesandgren) [inline easybuild/easyconfigs/m/M4/M4-1.4.18-GCCcore-9.1.0.eb]: https ?

File: easybuild/easyconfigs/z/zlib/zlib-1.2.11-GCCcore-9.1.0.eb [hunk 6]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'zlib'
+version = '1.2.11'
+
+homepage = 'http://www.zlib.net/'
```
Comment (akesandgren) [inline easybuild/easyconfigs/z/zlib/zlib-1.2.11-GCCcore-9.1.0.eb]: https ?

File: easybuild/easyconfigs/z/zlib/zlib-1.2.11-GCCcore-9.1.0.eb [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+description = """zlib is designed to be a free, general-purpose, legally unencumbered -- that is,
+ not covered by any patents -- lossless data-compression library for use on virtually any
+ computer hardware and operating system."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.1.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://zlib.net/fossils']
```
Comment (akesandgren) [inline easybuild/easyconfigs/z/zlib/zlib-1.2.11-GCCcore-9.1.0.eb]: https ?

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 10 out of 10 (8 easyconfigs in this PR)
hebbe-s1.int.private - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.14
See https://gist.github.com/4690ab002c2f4fd2ded39905a0b47800 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**